pages/dao/[symbol]/proposal/components/instructions/Dual/StakingOption.tsx

```diff
@@ -205,6 +251,31 @@ const StakingOption = ({
           error={formErrors['userPk']}
         />
       </Tooltip>
+      {baseMetadata && quoteMetadata && (
+        <>
+          <div className="p-3 border rounded-lg text-fgd-1 border-fgd-4 w-full">
+          {form.strike / form.lotSize * 10 ** (-quoteMetadata.decimals + baseMetadata.decimals)}
```

Comment:
Can we show the inverse too for ease/readability on puts

Reply:
changed it to show the exact quantities at exercise instead of ratio. So inverse shouldnt make sense anymore.

Reply:
Sure Total swap quantity works too